### DISCUSSION TITLE
how to get sigmet radar data in lat, lon, lev cooordinate and netcdf coordinte?

### DISCUSSION THREAD

**Discussion by amanii84** [ARM-DOE/pyart#1377] [General]
I get the netcdf file but it is in Cartesian coordinate x,y,z 
pyart.io.write_grid('file paths, grid) and lat, lon , lev as a new point written by using write_point_lon_lat_alt=True in the file
but not as default coordinate?
what should I do to convert  X, Y ,Z to lat, lon, lev

**Comment by zssherman** [comment]
Pulling in @mgrover1 as well, @amanii84 can you share the raw file? Is the raw file in Cartesian, x,y,z coordinates and your trying to convert it to lat lon, alt? Or are you trying to get a Cartesian grid with saved radar lat lon alt.

**Reply by amanii84** [reply]
I converted it to Cartesian grid X y z and I get the lat lon as a point not the default coordinate, I want to open it by ncview and I  will send the file as soon as possible.

**Reply by mgrover1** [reply]
I would covert to an xarray dataset,

```python
dataset = grid.to_xarray()
```
then use
```python
dataset.swap_dims({'x': 'lon', 'y': 'lat', 'z': 'alt'})
```
